app/Insightly/Resources/InsightlyOpportunityResource.php

```diff
@@ -72,6 +73,21 @@ public function updateStage(int $id, OpportunityStage $stage): void
         $this->insightlyClient->sendRequest($stageRequest);
     }
 
+    public function updateState(Integration $integration, int $id, OpportunityState $state): void
```

Comment:
If you add `updateState` in this pull request it is best to also make a test for it.